ms18-applicatie/Controllers/team-a/CalendarController.cs

```diff
@@ -0,0 +1,184 @@
+﻿using Google.Apis.Calendar.v3;
+using Google.Apis.Calendar.v3.Data;
+using Google.Apis.Services;
+using Microsoft.AspNetCore.Mvc;
+using ms18_applicatie.Models.team_a;
+using Google.Apis.Auth.OAuth2;
+
+namespace ms18_applicatie.Controllers.team_a
+{
+    [Route("Calendar")]
+    public class CalendarController : Controller
+    {
+        private readonly ILogger<HomeController> _logger;
+
+        private readonly CalendarSettings _calendarSettings;
+        private readonly CalendarService _calendarService;
+        public CalendarController(ILogger<HomeController> logger, IConfiguration configuration)
+        {
+            _logger = logger;
+
+            _calendarSettings =
+                configuration.GetSection("Calendar").Get<CalendarSettings>();
+
+            string[] scopes = new string[] { CalendarService.Scope.Calendar, CalendarService.Scope.CalendarEvents };
+            GoogleCredential credential;
+            using (var stream = new FileStream("./Controllers/team-a/service_accountInfo.json", FileMode.Open, FileAccess.Read))
+            {
+                credential = GoogleCredential.FromStream(stream)
+                    .CreateScoped(scopes);
+            }
+
+            //Create the Calendar service.
+            _calendarService = new CalendarService(new BaseClientService.Initializer()
+            {
+                HttpClientInitializer = credential,
+                ApplicationName = "project c",
+            });
+        }
+
+        [HttpDelete]
+        [Route("Event")]
+        public async Task<IActionResult> RemoveEvent(Calenders calenderName, string id)
+        {
+            var request = _calendarService.Events.Delete(GetCalenderId(calenderName), id);
+            await request.ExecuteAsync();
+            return new OkResult();
```

Comment:
er zou ook een mogelijkheid moeten zijn tot het terug geven van een badrequest in combinatie met een try/catch blok zodat de applicatie niet crashed. Dit geldt voor elk endpoint